### DISCUSSION TITLE
How do i configure to build multiarch docker images for windows and linux

### DISCUSSION THREAD

**Discussion by viceice** [goreleaser/goreleaser#2039] [Q&A] [answered]
Is there any sample how to build multiple docker images for windows and linux. And publish them as multiarch manifest to a registry?

ref: meltwater/drone-cache#149

**Comment by caarlos0** [comment, marked as answer]
should be the same as linux only images, just need to setup the platform et al

example of only linux: https://carlosbecker.com/posts/multi-platform-docker-images-goreleaser-gh-actions/

PS: not sure if you can create linux docker images from windows, but afaik you cant create windows docker images from linux, so that might be the issue here...

**Reply by viceice** [reply]
Yes, that's the issue. We like to build linux / windows multi arch images. Linuy only multi-arch isn't an issue. So you suggestion isn't a solution.

**Reply by vedantmgoyal9** [reply]
you can create linux-based docker images on windows using wsl2, which is supported in the windows-2025 github runners. however, i'm unsure how to run the docker cli within a wsl distribution or if it's even possible.

**Reply by vedantmgoyal9** [reply]
wait... you replied after whole 4 years 💀

**Reply by viceice** [reply]
> wait... you replied after whole 4 years 💀 

I got the resolved notification but missed the answer somehow 😔
